src/vstart.sh

```diff
@@ -1130,6 +1135,15 @@ EOF
     fi
 }
 
+start_exporter() {
+    echo "Starting Ceph exporter daemon..."
+
+    # Define socket directory for the exporter
+    # Start the exporter daemon 
+    prunb ceph-exporter --sock-dir "$CEPH_ASOK_DIR"
+    echo "Ceph exporter daemon started"
```

Comment:
I'd remove this message as redundant and even potentially confusing.  The daemon is spawned in the background, we don't wait for it to start (and don't even know whether it's going to start at all).
```suggestion
```